ajax/libs/overpass/3.0.2/overpass-webfont/overpass.css

```diff
@@ -0,0 +1,189 @@
+@font-face {
+  font-family: 'overpass';
+  src: url('overpass-thin.eot'); /* IE9 Compat Modes */
+  src: url('overpass-thin.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('overpass-thin.woff2') format('woff2'), /* Super Modern Browsers */
+       url('overpass-thin.woff') format('woff'), /* Pretty Modern Browsers */
+       url('overpass-thin.ttf')  format('truetype'); /* Safari, Android, iOS */
+       font-weight: 200;
+       font-style: normal;
+}
+
+@font-face {
+  font-family: 'overpass';
+  src: url('overpass-thin-italic.eot');
+  src: url('overpass-thin-italic.eot?#iefix') format('embedded-opentype'),
+       url('overpass-thin-italic.woff2') format('woff2'),
+       url('overpass-thin-italic.woff') format('woff'),
+       url('overpass-thin-italic.ttf')  format('truetype');
+       font-weight: 200;
+       font-style: italic;
+}
+
+
+@font-face {
+  font-family: 'overpass';
+  src: url('overpass-extralight.eot');
+  src: url('overpass-extralight.eot?#iefix') format('embedded-opentype'),
+       url('overpass-extralight.woff2') format('woff2'),
+       url('overpass-extralight.woff') format('woff'),
+       url('overpass-extralight.ttf')  format('truetype');
+       font-weight: 300;
+       font-style: normal;
+}
+
+@font-face {
+  font-family: 'overpass';
+  src: url('overpass-extralight-italic.eot');
+  src: url('overpass-extralight-italic.eot?#iefix') format('embedded-opentype'),
+       url('overpass-extralight-italic.woff2') format('woff2'),
+       url('overpass-extralight-italic.woff') format('woff'),
+       url('overpass-extralight-italic.ttf')  format('truetype');
+       font-weight: 300;
+       font-style: italic;
+}
+
+
+
+@font-face {
+  font-family: 'overpass';
+  src: url('overpass-light.eot');
+  src: url('overpass-light.eot?#iefix') format('embedded-opentype'),
+       url('overpass-light.woff2') format('woff2'),
+       url('overpass-light.woff') format('woff'),
+       url('overpass-light.ttf')  format('truetype');
+       font-weight: 400;
+       font-style: normal;
+}
+
+@font-face {
+  font-family: 'overpass';
+  src: url('overpass-light-italic.eot');
+  src: url('overpass-light-italic.eot?#iefix') format('embedded-opentype'),
+       url('overpass-light-italic.woff2') format('woff2'),
+       url('overpass-light-italic.woff') format('woff'),
+       url('overpass-light-italic.ttf')  format('truetype');
+       font-weight: 400;
+       font-style: italic;
+}
+
+
+
+@font-face {
+  font-family: 'overpass';
+  src: url('overpass-regular.eot');
+  src: url('overpass-regular.eot?#iefix') format('embedded-opentype'),
+       url('overpass-regular.woff2') format('woff2'),
+       url('overpass-regular.woff') format('woff'),
+       url('overpass-regular.ttf')  format('truetype');
+       font-weight: 500;
+       font-style: normal;
+}
+
+@font-face {
+  font-family: 'overpass';
+  src: url('overpass-italic.eot');
```

Comment:
Your files of `v3.0.2` seems different from source e.g. https://github.com/RedHatBrand/Overpass/blob/3.0.2/webfonts/overpass-webfont/overpass.css#L85
Would you refetch these files again?